src/main/webapp/WEB-INF/themes/desktop/sass/ApertaRxiv/vendor/kube/components/_typography.scss

```diff
@@ -20,7 +20,7 @@ a:hover {
 // Headings
 h1.title, h1, h2, h3, h4, h5, h6 {
 	font-family: $headings-font-family;
-	font-weight: bold;
+	font-weight: 600;
```

Comment:
we should probably just overwrite this as we agreed, let me check in the stuff i did when we talked last week.

Reply:
let me know if you are handling it.

Reply:
done.